Makes the pthread library link to libcxxtools.la by LIBADD, which means
that every program linking -lcxxtools will also link -lpthread (if pthread
is supported).

diff -u cxxtools-1.4.5_orig/src/Makefile.am cxxtools-1.4.5/src/Makefile.am
--- cxxtools-1.4.5_orig/src/Makefile.am	2007-10-25 10:19:25.000000000 +0000
+++ cxxtools-1.4.5/src/Makefile.am	2007-10-25 10:19:46.000000000 +0000
@@ -32,6 +32,6 @@
 	udpstream.cpp \
 	xmltag.cpp
 
-libcxxtools_la_LIBADD = @LIBLTDL@
+libcxxtools_la_LIBADD = @LIBLTDL@ $(PTHREAD_LIBS)
 libcxxtools_la_CXXFLAGS = $(PTHREAD_CFLAGS)
-libcxxtools_la_LDFLAGS = -version-info @sonumber@ $(PTHREAD_LIBS)
+libcxxtools_la_LDFLAGS = -version-info @sonumber@
